### ISSUE TITLE
A static class member is incorrectly reported as redefining an imported constant (F811)

### ISSUE THREAD

**Issue by asottile** [PyCQA/flake8#1095]
In GitLab by @rec1 on May 15, 2019, 06:02

*Please describe how you installed Flake8*

`$ pip install flake8` in a Python 3.6 virtualenv

*Please provide the exact, unmodified output of `flake8 --bug-report`*

flake8 --bug-report
{
  "dependencies": [
    {
      "dependency": "entrypoints",
      "version": "0.3"
    }
  ],
  "platform": {
    "python_implementation": "CPython",
    "python_version": "3.6.6",
    "system": "Darwin"
  },
  "plugins": [
    {
      "is_local": false,
      "plugin": "mccabe",
      "version": "0.6.1"
    },
    {
      "is_local": false,
      "plugin": "pycodestyle",
      "version": "2.5.0"
    },
    {
      "is_local": false,
      "plugin": "pyflakes",
      "version": "2.1.1"
    }
  ],
  "version": "3.7.7"
}

*Please describe the problem or feature*

flake8 incorrectly reports an F811 "redefinition of unused" when a static class member has the same name as a symbol imported from another module.

*If this is a bug report, please explain with examples (and example code) what you expected to happen and what actually happened.*

I expect the following Python code to get no flake8 warnings:

```python
from colors import COLORS  # noqa: F401


class Class:
    @property
    def COLORS(self):
        pass
```

Instead I get `./failure.py:5:5: F811 redefinition of unused 'COLORS' from line 1`

*Workaround.*

Add `# noqa: F811` to the end of the line with the reported warning.

**Comment by asottile** [comment]
In GitLab by @asottile on May 15, 2019, 06:08

flake8 is a framework and does not generate codes, please report this to the respective wrapped project (pyflakes)

**Comment by asottile** [comment]
In GitLab by @asottile on May 15, 2019, 06:08

closed

**Comment by asottile** [comment]
In GitLab by @rec1 on May 15, 2019, 06:36

Ah, sorry, I hadn't understood that.  Thanks for a great framework!

**Comment by asottile** [comment]
In GitLab by @rec1 on May 15, 2019, 06:47

See https://github.com/PyCQA/pyflakes/issues/445 in case anyone else ends up here.
